src/main/java/com/example/restapisample/ProfileController.java

```diff
@@ -0,0 +1,32 @@
+package com.example.restapisample;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import javax.validation.Valid;
+import java.net.URI;
+
+@Validated
+@RestController
+public class ProfileController {
+    @GetMapping("/profile")
+    public String getProfile(
+            @RequestParam(name = "name") String name,
+            @RequestParam(name = "day_of_birth") String day_of_birth) {
```

Comment:
クエリ文字列の命名規則とJavaの変数名の命名規則は分けて従ってほしいです。
```suggestion
            @RequestParam(name = "day_of_birth") String dayOfBirth) {
```
のようになります。

Reply:
承知しました、何事にも規則があるということが良く理解出来ました。
変更します。